### ISSUE TITLE
sharpersist unable to execute

### ISSUE THREAD

**Issue by lempeng** [BishopFox/sliver#720]
shapersist unable to execute

command:
sharpersist -t schtask -c "C:\Windows\System32\cmd.exe" -a "/c echo 123 >> c:\123.txt" -n "Some Task" -m add

result:
not execute.

expected:
execute

sliver server : ubuntu
sliver client : win 10

**Comment by moloch--** [comment]
I think you need something _like_:

```
sharpersist \"-t schtask -c 'C:\Windows\System32\cmd.exe' -a '/c echo 123 >> c:\123.txt' -n 'Some Task' -m add\"
```

I don't have a setup to test right now but can test later.

**Comment by lempeng** [comment]
not working

**Comment by lempeng** [comment]
tried this simple cmd:

`sharpersist \"-t schtask -m list\"`

But not working

**Comment by lempeng** [comment]
might be you can give me some idea to test

**Comment by lempeng** [comment]
`sliver (ALERT_CAREER) > sharpersist ' '-h

[*] sharpersist output:

[-] You specified duplicate switches. Check your command again.
`

**Comment by rkervella** [comment]
There was an issue in the way we parsed arguments, a leading space character was added to the argument string making some C# assemblies fail. It should be fixed by #722, I confirmed I'm able to run the following command without issues:

```shell
sharpersist '' -t service -c \"C:\Windows\System32\cmd.exe\" -a \"/c calc.exe\" -n \"Some Service\" -m add
```

**Comment by lempeng** [comment]
not really sure why i still not able to execute it

**Comment by cmprmsd** [comment]
Thanks @rkervella
I also stumbled over this issue as sharpersist was "eating up" spaces. 

With `sharpersist -- -c \"path\\with spaces\\bin.exe\" ...` it worked finally :+1: 

As -- has been implemented by grumble maybe it's possible to remove the need of escaping quotes?
At the moment the handling is quite intransparent and should be explained in the wiki entry for extensions, which we mentioned in https://github.com/BishopFox/sliver/issues/742.
